cmd/configuration/configuration.go

```diff
@@ -25,28 +25,33 @@ import (
 
 // Long descriptions are whitespace sensitive when generating docs using Sphinx.
 const (
-	initCmdShort = `Generates FlyteCTL config file in the user's home directory.`
-	initCmdLong  = `Creates a FlyteCTL config file in Flyte directory i.e ~/.flyte
+	initCmdShort = `Generates a Flytectl config file in the user's home directory.`
+	initCmdLong  = `Creates a Flytectl config file in Flyte directory i.e ~/.flyte.
 	
-Generates sandbox config. Flyte Sandbox is a fully standalone minimal environment for running Flyte. Read more about sandbox https://docs.flyte.org/en/latest/deployment/sandbox.html
-
+Generate Sandbox config:
 ::
 
  flytectl config init  
 
-Generates remote cluster config, By default connection is secure. Read more about the remote deployment https://docs.flyte.org/en/latest/deployment/index.html
+Flyte Sandbox is a fully standalone minimal environment for running Flyte. 
+Read more about Sandbox deployment :ref:` + "`here <deploy-sandbox-local>`" + `.
+
+Generate remote cluster config: 
 	
 ::
 
  flytectl config init --host=flyte.myexample.com
 
-Generates remote cluster config with insecure connection
+By default, the connection is secure. 
+Read more about remote deployment :ref:` + "`here <Deployment>`" + `.
+
+Generate remote cluster config with insecure connection:
```

Comment:
nit: not sure why "Generate" is getting rendered as "Generates". Can you please check?

Reply:
It's the same everywhere.